pool/routes/routes.ml

```diff
@@ -385,7 +385,7 @@ module Admin = struct
       let assignments =
         let open Experiments.Assignment in
         [ get "" ~middlewares:[ Access.index ] index
-        ; get "deleted" ~middlewares:[ Access.deleted ] deleted
+        ; get "deleted" ~middlewares:[ Access.delete ] deleted
```

Comment:
This middleware does not work. The route `admin/experiments/983a1f96-5862-4b86-91fa-0f77d2e02ed7/assignments/deleted` does not contain an assignment_id. It is a get request to show all deleted assignments.

IMO we can use the `index` access.

Reply:
refactored